src/SprykerEco/Yves/Payone/PayoneFactory.php

```diff
@@ -84,10 +88,18 @@ public function createSecurityInvoiceSubForm(): SubFormInterface
         return new SecurityInvoiceSubForm();
     }
 
+    /**
+     * @return \SprykerEco\Yves\Payone\Form\KlarnaSubForm
+     */
+    public function createKlarnaSubForm(): SubFormInterface
+    {
+        return new KlarnaSubForm();
+    }
+
     /**
      * @return \SprykerEco\Yves\Payone\Form\DataProvider\InvoiceDataProvider
      */
-    public function createInvoiceSubFormDataProvider()
+    public function createInvoiceSubFormDataProvider(): InvoiceDataProvider
```

Comment:
interface

Reply:
Rolled back for avoiding return type conflicts for customers